ambari-server/src/main/resources/stack-hooks/after-INSTALL/scripts/params.py

```diff
@@ -68,36 +72,36 @@
 security_enabled = get_cluster_setting_value('security_enabled')
 
 #java params
-java_home = config['ambariLevelParams']['java_home']
+java_home = execution_command.get_java_home()
 
 #hadoop params
-hdfs_log_dir_prefix = config['configurations']['hadoop-env']['hdfs_log_dir_prefix']
-hadoop_pid_dir_prefix = config['configurations']['hadoop-env']['hadoop_pid_dir_prefix']
-hadoop_root_logger = config['configurations']['hadoop-env']['hadoop_root_logger']
+hdfs_log_dir_prefix = module_configs.get_property_value(module_name, 'hadoop-env', 'hdfs_log_dir_prefix')
+hadoop_pid_dir_prefix = module_configs.get_property_value(module_name, 'hadoop-env', 'hadoop_pid_dir_prefix')
+hadoop_root_logger = module_configs.get_property_value(module_name, 'hadoop-env', 'hadoop_root_logger')
 
 jsvc_path = "/usr/lib/bigtop-utils"
 
-hadoop_heapsize = config['configurations']['hadoop-env']['hadoop_heapsize']
-namenode_heapsize = config['configurations']['hadoop-env']['namenode_heapsize']
-namenode_opt_newsize = config['configurations']['hadoop-env']['namenode_opt_newsize']
-namenode_opt_maxnewsize = config['configurations']['hadoop-env']['namenode_opt_maxnewsize']
-namenode_opt_permsize = format_jvm_option("/configurations/hadoop-env/namenode_opt_permsize","128m")
-namenode_opt_maxpermsize = format_jvm_option("/configurations/hadoop-env/namenode_opt_maxpermsize","256m")
+hadoop_heapsize = module_configs.get_property_value(module_name, 'hadoop-env', 'hadoop_heapsize')
```

Comment:
When this hook gets called for say Zookeeper-install, the module-name will be zookeeper. How will the hadoop related variables be resolved in this case? Problem is namenode, hdfs, hadoop related variables in this common hooks code.
Is this something the instance manager needs to look at?
@jayush , @d0zen1

Reply:
At this time, module_name is not used, if you see module_config.py, it is not used and just a placeholder.